scripts/report_coverage.sh

```diff
@@ -0,0 +1,56 @@
+mkdir -p coverage/merged
```

Comment:
need to check where it is executing (PWD)
also, probably add PWD to each accessed file and properly locate them